internal/publish/database/exposure.go

```diff
@@ -189,7 +198,16 @@ func generateExposureQuery(criteria IterateExposuresCriteria) (string, []interfa
 		q += fmt.Sprintf(" AND local_provenance = $%d", len(args))
 	}
 
-	q += " ORDER BY created_at"
+	if criteria.OnlyTravelers {
+		args = append(args, true)
+		q += fmt.Sprintf(" AND traveler = $%d", len(args))
```

Comment:
Isn't this a boolean? Why would it be equal to an int?

Reply:
it's not, it putting in a bind for $XX where XX is the current length of args.
args itself is bound as query params